.github/config/spellcheck.yml

```diff
@@ -0,0 +1,22 @@
+matrix:
+  - name: Markdown
+    aspell:
+      lang: en
+      ignore-case: true
+      mode: markdown
+    dictionary:
+      wordlists:
+        - .github/config/wordlist.txt
+      encoding: utf-8
+    pipeline:
+      - pyspelling.filters.markdown:
+          markdown_extensions:
+            - pymdownx.superfences
+      - pyspelling.filters.html:
+          comments: false
+          ignores:
+            - code
+            - pre
+    sources:
+      - '**/*.md'
```

Comment:
We also have `html` sources if I am not mistaken (e.g. https://github.com/open-component-model/ocm-website/blob/main/layouts/index.html). Would it be possible to check them? Or would the `html`-syntax be too troublesome.

Reply:
I'm not too convinced that the spell checker really makes sense for html pages and propose to keep the check for markdown only.